shell/components/nav/WindowManager/__tests__/ContainerShell.test.ts

```diff
@@ -277,10 +279,10 @@ describe('component: ContainerShell', () => {
     eventConnected();
     eventMessage({ detail: { data: `3${ errorMessage }` } });
 
-    expect(consoleError.mock.calls[0][0]).toBe(errorMessage);
+    expect(consoleError.mock.calls[0][0]).toContain(errorMessage);
```

Comment:
I opted for `toContain` because I didn't perceive the actual content on the error message mattering as much as asserting that the error message exists. If we don't believe that this will be accurate enough, we can look into better mocks for dayjs and i18n so that we can assert that the content of the message matches the new format.